src/types/component.go

```diff
@@ -122,5 +116,5 @@ type ZarfDataInjection struct {
 // ZarfImport structure for including imported zarf components
 type ZarfComponentImport struct {
 	ComponentName string `yaml:"name,omitempty"`
-	Path          string `yaml:"path"`
+	Path          string `yaml:"path" jsonschema:"pattern=^(?!.*###ZARF_PKG_VAR_).*$"`
```

Comment:
One of the things we've added elsewhere in the codebase for potentially non-obvious regex is a link in the comments to a regex tool with some validations to show how it works.  

https://github.com/defenseunicorns/zarf/blob/v0.20.0/src/internal/utils/image.go#L5

I realize that we haven't done this for any other pattern matching in the schemas and that the schema doc generator links you to a regex test site, but if we don't have a link like above, could you put a small comment aabout what's happening/why for someone who isn't familiar with regex?

Reply:
Added a link

Reply:
I must be going blind...
<img width="1346" alt="Screen Shot 2022-07-28 at 1 11 17 PM" src="https://user-images.githubusercontent.com/882485/181607908-d1ff06a1-ba20-4a1e-aba3-467cc7b3ee92.png">
.

Reply:
It's there for me. Since you're seeing the `Changed since last view` label you probably just need to refresh

Reply:
![image](https://user-images.githubusercontent.com/3977569/181613431-80460421-d318-4ced-b564-9f05ee22975d.png)
It is there for me too.